### PR TITLE
parany.7.0.0: test adding dune lower bound version constraint

### PR DESCRIPTION
**Pull request by UnixJunkie**
I think it should be there.

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
There doesn't seem to be any `dune-project` file to back it up, could you think about it for future versions?

Also, looking at the only `dune` file, it doesn't seem to require 1.6.0, could you double check? https://github.com/UnixJunkie/parany/blob/f5f40d5a63ae5f9fe51b3dd462aa5c7a59d46015/src/dune

File: packages/parany/parany.7.0.0/opam [hunk 1]
```diff
@@ -7,7 +7,7 @@ bug-reports: "https://github.com/UnixJunkie/parany/issues"
 dev-repo: "git+https://github.com/UnixJunkie/parany.git"
 depends: [
   "ocaml" {>= "4.03.0"}
-  "dune"
+  "dune" {>= "1.6.0"}
```
Comment (kit-ty-kate) [inline packages/parany/parany.7.0.0/opam]:
```suggestion
  "dune" {< "2.0"}
```
Oh, looking at the error in #15398, I think this requires a upper-bound constraint instead
```
#=== ERROR while compiling parany.7.0.0 =======================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.09.0 | git+file:///home/opam/opam-repository
# path                 ~/.opam/4.09/.opam-switch/build/parany.7.0.0
# command              ~/.opam/4.09/bin/dune build -p parany -j 2
# exit-code            1
# env-file             ~/.opam/log/parany-25-42ef32.env
# output-file          ~/.opam/log/parany-25-42ef32.out
### output ###
# Info: Creating file dune-project with this contents:
# | (lang dune 2.0)
# | (name parany)
# File "src/dune", line 15, characters 2-36:
# 15 |   (action
# 16 |   (run ./test.exe 16 800)))
# Error: 'action' was deleted in version 2.0 of the dune language. Use a rule
# stanza with the alias field instead

- Info: Creating file dune-project with this contents:
- | (lang dune 2.0)
- | (name parany)
- File "src/dune", line 15, characters 2-36:
- 15 |   (action
- 16 |   (run ./test.exe 16 800)))
- Error: 'action' was deleted in version 2.0 of the dune language. Use a rule
- stanza with the alias field instead
```

**Comment by UnixJunkie** [comment]
the dune-project file was listed in the .gitignore; that's why it's not in the tarball.
I will send a minor update.

**Comment by mseri** [comment]
Should we merge this?

**Comment by kit-ty-kate** [comment]
Yes it is relevant for https://github.com/ocaml/opam-repository/pull/15398. Thanks!

**Pull request merged**